content/01.abstract.md

```diff
@@ -7,9 +7,9 @@ Since functional signatures of cancer mutation have been identified across many
 The TCGA Pan-Cancer Atlas contains genetic alteration data including somatic mutations and copy number variants (CNVs), as well as several -omics data types.
 From TCGA, we focus on RNA sequencing, DNA methylation arrays, reverse phase protein arrays (RPPA), microRNA, and somatic mutational signatures as -omics readouts.
 
-Across a collection of cancer-associated genetic alterations, RNA sequencing and DNA methylation were the most effective predictors of alteration state.
-Surprisingly, we found that for most alterations, they were approximately equally effective predictors.
-The target gene was the primary driver of performance, rather than the data type, and there was little difference between the top data types for the majority of genes.
-We also found that combining data types into a single multi-omics model often provided little or no improvement in predictive ability over the best individual data type.
+Across a collection of genes recurrently mutated in cancer, RNA sequencing and DNA methylation were the most effective predictors of mutation state.
+Surprisingly, we found that for most mutated genes, they were approximately equally effective predictors.
+Performance was more variable between mutations than it was between data types for the same mutation, and there was little difference between the top data types for the majority of genes.
```

Comment:
This new version is so much easier for me to follow! I think it communicates the importance of your findings right off the bat!